packages/browser-agent-core/common/aggregate/aggregator.js

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { SharedContext } from '../context/shared-context'
+import { mapOwn } from '../util/map-own'
+
+export class Aggregator extends SharedContext {
```

Comment:
Some clarity in the names of these functions has been lost because we've named the methods what they were named in the original export. Maybe we can review this file to rename and re-add some old comments:

https://github.com/newrelic/newrelic-browser-agent/blob/487a282f92dd02a32b2b94d41bd7b718f09ff30c/agent/aggregator.js#L11-L15

Reply:
Btw my 2 cents from working on tests: it did not make sense as reader for the code to rename a specialized function with a generalized version of its name. For example, don't expose "storeMetricUnderAgentEE" as "storeMetric". It's non-descript, vague, and potentially misleading. Expose methods as they are, or at least abbreviate them without losing context, e.g. "storeMetricAgtEE".